include/BSMPT/utility/const_velocity_spline.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * vev_spline.cpp
+ *
+ * cubic spline interpolation for path in VEV space using constant speed (in
+ * particular = 1)
+ *
+ * We use the lib spline.h to make a spline in each direction, the parameter is
+ * the linear length between the points (not important as long as it
+ * monotonically increasing) Then we make a thoroughly integration (Simpson 3/8
+ * rule) to convert from linear length to spline length, and vice-versa.
+ * Interpolate using a spline
+ *
+ */
+
+#include <BSMPT/utility/Logger.h> // for Logger Class
+#include <BSMPT/utility/spline.h>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+#ifndef INCLUDE_BSMPT_GravitationalWaves_GWUtils_ConstantVelocitySpline_H_
+#define INCLUDE_BSMPT_GravitationalWaves_GWUtils_ConstantVelocitySpline_H_
+
+namespace cvspline
+{
+double length_vector(std::vector<double> x0, int dim);
+
+std::vector<std::vector<double>> transpose(std::vector<std::vector<double>> &A);
```

Comment:
Is the missing const reference here by choice or do you modify the input? Some documentation would be nice

Reply:
It is missing. Corrected and documentation added.